app/controllers/readings_controller.rb

```diff
@@ -14,30 +16,25 @@ def index
     end
   end
 
-  # GET /readings/1 or /readings/1.json
-  def show
-  end
-
   # GET /readings/new
   def new
     @reading = Reading.new
+    authorize @reading, :new? 
```

Comment:
refactor since you already have `before_action :authorize_reading, only: %i[show edit update destroy]` on line 4